<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajouter_Assrances</class>
 <widget class="QDialog" name="Ajouter_Assrances">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajout_assurance_pb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ajout_id_assurance">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ajout_nom_assurance">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ajout_type_assurance">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une assurance</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
